testutil/base.go

```diff
@@ -11,6 +11,12 @@ import (
 	dtypes "github.com/ovrclk/akash/x/deployment/types"
 )
 
+func init() {
+	config := sdk.GetConfig()
+	config.SetBech32PrefixForAccount(CoinDenom, CoinDenom)
+	config.Seal()
```

Comment:
this is only going to cause problems.  let's not be hesitant to back it out.

Reply:
Agreed, I'm not sure what the better solution is though.

Reply:
In principal the tests shouldn't care about the coin denomination.